### PR TITLE
Add slide and video link to mentors table

### LINKED ISSUE #581: Add slide link as a database column to mentor users
To instantiate a set of CSS carousels by advisor name.

### PR DESCRIPTION
**Pull request by potatowagon**
resolves #581

### CODE REVIEW

**Comment by knmnyn** [comment]
LGTM.  No controller or view code; just model changes.  Shouldn't be a problem.

**Comment by chowyb** [comment]
Yep, seems okay.

@knmnyn just to check, do I need to manually run "rails db:migrate" or any similar commands on the production server after merging this PR? It looks like the bash script already handles that, but I'm not too sure.

**Comment by knmnyn** [comment]
@chowyb I think as you said the deploy script auto runs db:migrate.  You may see some errors or warnings but I don't think they are due to this PR.

Thanks!

**Comment by zhaojin42** [comment]
@potatowagon, as mentioned in #581, we are planning to add only a slide link to the mentors table. That slide is meant to be a self-introduction of the mentor. 

The place where both slide and and video links are needed is the Milestone 0 for the teams.

Please make the necessary changes and submit another PR.

**Comment by potatowagon** [comment]
teams-db
mentors-db

**Comment by potatowagon** [comment]
@zhaojin42 changes made

File: db/schema.rb [hunk 1]
```diff
@@ -472,7 +475,6 @@
     t.string   "blog_link",              default: ""
     t.integer  "program_of_study",       default: 0
     t.text     "self_introduction",      default: ""
-    t.string   "slack_id",               default: ""
```
Comment (chowyb) [inline db/schema.rb]: Just wondering if you happened to know why this line was removed? The migrations introduced in this PR doesn't seem to touch this at all, so I'm not sure what caused this to happen.

Comment (potatowagon) [reply db/schema.rb]: hmm thats true, i dont know how it got removed. I will add it back, thanks for spotting this

**Comment by potatowagon** [comment]
@chowyb resolved

**Pull request merged**